security/oauth/manager/config.go

```diff
@@ -0,0 +1,41 @@
+package manager
+
+import (
+	"net/url"
+	"time"
+
+	"golang.org/x/oauth2/clientcredentials"
+)
+
+type Endpoint struct {
+	TokenURL string `envconfig:"TOKEN_URL" env:"TOKEN_URL"`
+}
+
+type Config struct {
```

Comment:
I discussed about the comment with @ondrejtomcik and for now we want to use env variables for now. But we want to add option to load configuration from file and it will be implemented in services, not in kit.